pkgs/servers/monitoring/plugins/zfs.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, zfs
+, sudo
+}:
+
+let
+  n="check_zfs";
+in stdenv.mkDerivation {
+  pname = "check-zfs";
```

Comment:
Just a suggestion: I think it's better to leave it as `check_zfs` and not rename it.